package.json

```diff
@@ -91,7 +94,8 @@
       "prettier --write"
     ],
     "{,!(node_modules)/**/}*.{css,scss}": [
-      "prettier --write"
+      "prettier --write",
+      "stylelint --cache --fix"
```

Comment:
Were you able to test this ?

Reply:
yes it works, i tried committing with violation and it auto resolves for me